src/main/java/seedu/address/logic/commands/PriorityCommand.java

```diff
@@ -65,7 +65,7 @@ public CommandResult execute(Model model) throws CommandException {
                 Person editedPerson = new Person(
                         person.getName(), person.getPhone(), person.getEmail(),
                         person.getNric(), person.getAddress(), person.getDateOfBirth(),
-                        person.getGender(), person.getTags(), person.getAppointments());
+                        person.getGender(), person.getTags(), priority, person.getAppointments());
```

Comment:
The getTags is for what ah?

Reply:
When we amend a particular parameter of the Person , we create a new instance of the person as the variables are immutable. If you would like to assign/change the priority of an existing person, you must create a new instance of it with the same variables as it was before with the exception of the variable you are changing, hence the getters. Hope this clarifies.